org-code-javabuilder/lib/src/main/java/org/code/javabuilder/TestRunner.java

```diff
@@ -1,86 +1,20 @@
 package org.code.javabuilder;
 
-import java.lang.reflect.Method;
-import java.net.URLClassLoader;
-import java.util.ArrayList;
 import java.util.List;
-import org.code.javabuilder.util.ProjectLoadUtils;
-import org.code.protocol.InternalErrorKey;
 import org.code.protocol.OutputAdapter;
-import org.code.protocol.StatusMessage;
 import org.code.protocol.StatusMessageKey;
-import org.code.validation.support.NeighborhoodTracker;
-import org.code.validation.support.ValidationProtocol;
-import org.junit.platform.commons.PreconditionViolationException;
-import org.junit.platform.engine.discovery.ClassSelector;
-import org.junit.platform.engine.discovery.DiscoverySelectors;
-import org.junit.platform.launcher.Launcher;
-import org.junit.platform.launcher.LauncherDiscoveryRequest;
-import org.junit.platform.launcher.LauncherSession;
-import org.junit.platform.launcher.TestPlan;
-import org.junit.platform.launcher.core.LauncherDiscoveryRequestBuilder;
-import org.junit.platform.launcher.core.LauncherFactory;
 
 /** Runs all tests for a given set of Java files */
-public class TestRunner implements CodeRunner {
-  private final List<JavaProjectFile> javaFiles;
-  private final JavabuilderTestExecutionListener listener;
-  private final OutputAdapter outputAdapter;
+public class TestRunner extends BaseTestRunner {
```

Comment:
I'm a touch concerned that "TestRunner" will sound too much like an internal util. Maybe something like "UserTestRunner" would help differentiate from our tests?

Reply:
sure I can update it